osu.Game/Rulesets/Edit/Layers/Selection/SelectionOverlay.cs

```diff
@@ -15,15 +18,13 @@ namespace osu.Game.Rulesets.Edit.Layers.Selection
     /// <summary>
     /// A box which encloses <see cref="DrawableHitObject"/>s.
     /// </summary>
-    public class CaptureBox : VisibilityContainer
+    public class SelectionOverlay : VisibilityContainer

```

Comment:
does this change name in the future PR? what does it become, if so (if it does, i'd prefer not renaming it here).

Reply:
It remains as SelectionOverlay

Reply:
Seeing as the "future PR" is not PR'd yet, consider this as it is within the context of this PR, plz.

Reply:
you said in the description to ignore naming :| which one?

Reply:
removed from pr description